msal/build.gradle

```diff
@@ -74,6 +80,18 @@ android {
             outputFileName = fileName
         }
     }
+
+    testOptions {
+        unitTests.all {
+            exclude 'com/microsoft/identity/client/robolectric/tests/AcquireTokenNetworkTest.class'
+            exclude 'com/microsoft/identity/client/robolectric/tests/B2CTest.class'
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
```

Comment:
We have to be careful changing source compatibility here - this can cause issues for downstreamers of the library who are still using 1.7.

At a _minimum_, if you need this change
- Run it by immediate partners (Company Portal, Authenticator, Teams)
- Apply this to `common`, `adal`, and `android-complete` too

Reply:
Addressed.

That change has been removed as it was not needed after refactor.